packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -252,3 +252,42 @@ class ByType extends SerializableFinder {
     return new ByType(json['type']);
   }
 }
+
+
+/// A Flutter driver command that retrieves a semantics id using a specified finder.
```

Comment:
mention here as well that it might be the semantics information from an ancestor?